BookReaderTemplate/BookNavigator/BookNavigator.js

```diff
@@ -0,0 +1,332 @@
+import { html, LitElement } from 'lit-element';
```

Comment:
Let's place these in `src/`; code always lives in the `src` directory here. (e.g. `src/BookNavigator`, `src/iaux-foo-bar`). We'll at some point in the future want to remove the `js` directory, since things are organized by structural component instead of by file type :+1: (If that causes issues, we can also do e.g. `src/templates/BookNavigator`)

Reply:
updated now